### DISCUSSION TITLE
[Idea] New branch policy

### DISCUSSION THREAD

**Discussion by fulldecent** [PrestaShop/PrestaShop#35954] [Ideas]
### The problem

Currently GitHub branches are maintained as 8.0.x and 8.1.x.

But 8.0 is not supported, source:

> Supported Versions
> Security updates will typically only be applied to the latest release.
> 
> https://github.com/PrestaShop/PrestaShop/security

### The solution

It will be better if only a branch `8.x.x` (or `8`) is maintained because it guarantees compatibility to the next breaking change.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
Hello @fulldecent branch 8.0.x is not maintained anymore. It will not give birth to new versions.

The latests commits in 8.0.x branch are related to the CI, not the codebase
https://github.com/PrestaShop/PrestaShop/commits/8.0.x/
